src/ResourceManager/IotHub/Commands.IotHub/help/Set-AzureRmIotHub.md

```diff
@@ -0,0 +1,307 @@
+---
+external help file: Microsoft.Azure.Commands.IotHub.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Set-AzureRmIotHub
+
+## SYNOPSIS
+Updates the properties of an IotHub.
+
+## SYNTAX
+
+### UpdateSku
+```
+Set-AzureRmIotHub [-ResourceGroupName] <String> [-Name] <String> -SkuName <PSIotHubSku> [-Units <Int64>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### UpdateEventHubEndpointProperties
+```
+Set-AzureRmIotHub [-ResourceGroupName] <String> [-Name] <String> -EventHubRetentionTimeInDays <Int64> [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### UpdateFileUploadProperties
+```
+Set-AzureRmIotHub [-ResourceGroupName] <String> [-Name] <String> [-FileUploadStorageConnectionString <String>]
+ [-FileUploadContainerName <String>] [-FileUploadSasUriTtl <TimeSpan>] [-FileUploadNotificationTtl <TimeSpan>]
+ [-FileUploadNotificationMaxDeliveryCount <Int32>] -EnableFileUploadNotifications <Boolean> [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### UpdateCloudToDeviceProperties
+```
+Set-AzureRmIotHub [-ResourceGroupName] <String> [-Name] <String> -CloudToDevice <PSCloudToDeviceProperties>
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### UpdateOperationsMonitoringProperties
+```
+Set-AzureRmIotHub [-ResourceGroupName] <String> [-Name] <String>
+ -OperationsMonitoringProperties <PSOperationsMonitoringProperties> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates the properties of an IotHub.
+
+## EXAMPLES
+
+### Example 1 Update the sku
+```
+PS C:\> Set-AzureRmIotHub -ResourceGroupName "myresourcegroup" -Name "myiothub" -SkuName S1 -Units 5
+```
+Update the sku to S1 and units to 5 for the IotHub named "myiothub"
+
+### Example 2 Update the eventhub properties
+```
+PS C:\> Set-AzureRmIotHub -ResourceGroupName "myresourcegroup" -Name "myiothub" -EventHubRetentionTimeInDays 4
+```
+Update the retention time in days to 4 for both the telemetry and operationsmonitoringevents events for the IotHub named "myiothub"
+
+## PARAMETERS
+
+### -CloudToDevice
+CloudToDevice
+
```

Comment:
Need an informative description for each parameter.  This applies to all help files